pkg/manifests/manifests.go

```diff
@@ -1129,15 +1129,24 @@ func (f *Factory) PrometheusK8s(host string, grpcTLS *v1.Secret, trustedCABundle
 			URL:         f.config.TelemeterClientConfig.TelemeterServerURL,
 			BearerToken: base64.StdEncoding.EncodeToString(compositeToken),
 			QueueConfig: &monv1.QueueConfig{
-				Capacity: 10000,
-				// Default: 100
-				MaxSamplesPerSend: 1000,
-				// Default: 1000
-				MaxShards: 100,
-				// Default: 30ms
-				MinBackoff: "500ms",
-				// Default: 100ms
-				MaxBackoff: "2s",
+				// Amount of samples to load from the WAL into the in-memory
+				// buffer before waiting for samples to be sent successfully
+				// and then continuing to read from the WAL.
+				Capacity: 30000,
+				// Should we accumulate 10000 samples before the batch send
```

Comment:
You mean less than 1000 samples?

Reply:
The comment refers to the `MaxSamplesPerSend`, which is set to 10000.